.github/workflows/ci.yml

```diff
@@ -34,42 +35,33 @@ jobs:
 
       - name: lint
         run: |
-          go install honnef.co/go/tools/cmd/staticcheck@v0.4.7 &&
-          go install golang.org/x/tools/cmd/goimports@v0.24.0 &&
-          $HOME/go/bin/staticcheck &&
-          make vet &&
-          make check-gofmt
+          just lint format-check
 
   test:
-     runs-on: ubuntu-latest
-     strategy:
-       matrix:
-         go:
-           - "1.23"
-           - "1.22"
-           - "1.21"
-           - "1.20"
-           - "1.19"
-           - "1.18"
-           - "1.17"
-           - "1.16"
-           - "1.15"
-     name: "Test: go v${{ matrix.go }}"
-     steps:
-       - uses: actions/checkout@v2
-       - name: Setup go
-         uses: actions/setup-go@v1
-         with:
-           go-version: ${{ matrix.go }}
-       - uses: stripe/openapi/actions/stripe-mock@master
-       - name: Test
-         run: make ci-test
-       - name: Coveralls
-         run: make coverage && make coveralls
-         if: matrix.go == '1.16'
-         env:
-           COVERALLS_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-           COVERALLS_FLAG_NAME: Go-${{ matrix.go }}
+    runs-on: ubuntu-latest
```

Comment:
this block is mostly the same except coveralls is gone and `just` is used in favor of `make`. Also, prettier shifted it over (it was intended by 5 spaces?)